apps/judicial-system/backend/src/app/modules/verdict/guards/verdictExistGuard.guard.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  BadRequestException,
+  CanActivate,
+  ExecutionContext,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common'
+
+@Injectable()
+export class VerdictExistGuard implements CanActivate {
+  async canActivate(context: ExecutionContext): Promise<boolean> {
+    const request = context.switchToHttp().getRequest()
```

Comment:
You can skip this part. The case is verified in another guard and no need to repeat it.